skimage/feature/blob.py

```diff
@@ -321,7 +404,7 @@ def blob_doh(image, min_sigma=1, max_sigma=30, num_sigma=10, threshold=0.01,
 
     Parameters
     ----------
-    image : ndarray
+    image : 2D ndarray
```

Comment:
@emmanuelle the Hessian is now nD, so maybe you want to update this also? Would it be difficult? I wouldn't hold up this PR for it but it would be nice if you have some time! =)